addons/base_gengo/models/res_company.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+
+
+class ResCompany(models.Model):
+    _name = "res.company"
+    _inherit = "res.company"
+
+    gengo_private_key = fields.Text(string="Gengo Private Key",
```

Comment:
Remove the string parameter here, as the ORM will generate a capitalized version of the field name by default